packages/net-stubbing/lib/server/driver-events.ts

```diff
@@ -84,8 +93,10 @@ export async function onNetEvent (opts: OnNetEventOpts): Promise<any> {
   debug('received driver event %o', { eventName, args })
 
   switch (eventName) {
-    case 'route:added':
-      return _onRouteAdded(state, getFixture, <NetEventFrames.AddRoute>frame)
+    case 'create:route':
+      return _createRoute(state, getFixture, <NetEventFrames.CreateRoute>frame)
+    case 'set:route:disabled':
```

Comment:
ended up not going this route but i figure we may as well keep the method kicking around for prototyping cy intercept overrides